### PR TITLE
Add Security Magboots

### PR DESCRIPTION
**Pull request by ProphetCrow**
## About the PR
Adds Security Mag Boots

## Why / Balance
Gives security EVA magboots and slip resistance, 20% movement speed penalty when activated , which is statistically identical to engineering magboots.

## Technical details
Added files, adjusted suit_storage yml file in main prototypes catalog

## Media
https://cdn.discordapp.com/attachments/1010560161888350340/1261395908130639942/Delta-v_2024-07-12_14-49-55.mp4?ex=6692cdfc&is=66917c7c&hm=696963dad70232ebe0711f288bfe09cde1859c071425f1bd0705e0ccda78ce97&
/ https://discord.com/channels/968983104247185448/1010560161888350340/1261395910152163378

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
None

**Changelog**

:cl: Crow
- add: Security Magboots

### CODE REVIEW

File: Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/7e4e9d432d88981fb9bb463970c5b98ce85c0abe",
```
Comment (DangerRevolution) [inline Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: TGStation does not have Security Magboots; nor are they in this commit.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Shoes/magboots.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - type: Magboots
+    toggleAction: ActionToggleMagbootsSec
+    
+- type: entity
+  id: ActionToggleMagbootsSec
+  parent: ActionBaseToggleMagboots
+  components:
+  - type: InstantAction
+    icon: { sprite: DeltaV/Clothing/Shoes/Boots/magboots-security.rsi, state: icon }
+    iconOn: DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/icon-on.png
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Shoes/magboots.yml]:
```suggestion
    iconOn: DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/icon-on.png

```

**Review by DangerRevolution — COMMENTED** [review]
Ignoring that these are **INSANELY** unbalanced; and will be a Security DREAM.

Two issues, test fail aside :)

Changelog also needs fixing; and there are no breaking changes here.

File: Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml [hunk 3]
```diff
@@ -141,6 +141,7 @@
         - id: OxygenTankFilled
         - id: ClothingOuterHardsuitCombatOfficer # DeltaV - ClothingOuterHardsuitSecurity replaced in favour of security combat hardsuit.
         - id: ClothingMaskBreath
+        - id: ClothingShoesBootsMagSec
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml]:
```suggestion
        - id: ClothingShoesBootsMagSec # Delta-V : Add Security Magboots
```

File: Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml [hunk 4]
```diff
@@ -200,6 +201,7 @@
         - id: JetpackSecurityFilled
         - id: ClothingOuterHardsuitCombatHoS # DeltaV - ClothingOuterHardsuitSecurityRed replaced in favour of head of security's advanced combat hardsuit.
         - id: ClothingMaskGasSwat
+        - id: ClothingShoesBootsMagSec
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml]:
```suggestion
        - id: ClothingShoesBootsMagSec # Delta-V : Add Security Magboots
```

File: Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml [hunk 5]
```diff
@@ -214,6 +216,7 @@
         - id: OxygenTankFilled
         - id: ClothingOuterHardsuitCombatWarden # DeltaV - ClothingOuterHardsuitWarden replaced in favour of warden's riot combat hardsuit.
         - id: ClothingMaskBreath
+        - id: ClothingShoesBootsMagSec
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml]:
```suggestion
        - id: ClothingShoesBootsMagSec # Delta-V : Add Security Magboots
```

File: Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml [hunk 6]
```diff
@@ -275,6 +278,7 @@
       - id: OxygenTankFilled
       - id: ClothingOuterHardsuitCombatCorpsman
       - id: ClothingMaskBreath
+      - id: ClothingShoesBootsMagSec
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml]:
```suggestion
      - id: ClothingShoesBootsMagSec # Delta-V : Add Security Magboots
```

**Comment by VMSolidus** [comment]
> Ignoring that these are **INSANELY** unbalanced; and will be a Security DREAM.
> 
> Two issues, test fail aside :)
> 
> Changelog also needs fixing; and there are no breaking changes here.

The hell do you mean unbalanced?
Security hardsuit lockers just spawn with Engineering Magboots inside. All hardsuit lockers other than the CE locker do. Security officers are forced to wear orange magboots.

File: Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json (hunk 1 above)
Comment (ProphetCrow) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: the original sprite is theirs, I just colored over it. I guess fair use applies?

Comment (Timemaster99) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: > the original sprite is theirs, I just colored over it. I guess fair use applies?

If that's the case, add that to the attribution. "Original sprite from _____, edited by ______"

Comment (DangerRevolution) [inline Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]:
```suggestion
  "copyright": "Taken from Engineering Magboots from tgstation at commit https://github.com/tgstation/tgstation/commit/7e4e9d432d88981fb9bb463970c5b98ce85c0abe , edited by ProphetCrow (Github)",

```

**Comment by WarMechanic** [comment]
the sprite is nice :)

**Comment by BlitzTheSquishy** [comment]
As a sec main

Tie it to research rather than being round start

cause like engi always wearing theirs, sec will do the same - mitigate powergaming

**Comment by TadJohnson00** [comment]
Is this PR still being worked on or is it abandoned @ProphetCrow ?

**Comment by deltanedas** [comment]
> Tie it to research

they can already ask epi to make researched magboots

**Comment by Samsterious** [comment]
While I think Security should have access to magboots for EVA, we all know every SecOff is going to speedrun grabbing these out of the lockers if they are available, and that completely defeats one of the most effective ways of dealing with Security on your tail without harming them. This PR _makes sense_, but I also think it introduces a massive balance issue that far outweighs that fact.

**Comment by TadJohnson00** [comment]
Going to assume this is abandoned, no activity after two months and no reply after two weeks.

**Pull request closed without merging**